assisted-events-scrape/repositories/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .event_repository import EventRepository
+from .cluster_repository import ClusterRepository
```

Comment:
Need to declare `__all__` for defining what symbols in a module will be exported